packages/react-devtools-shared/src/devtools/views/Profiler/CommitFlamegraph.js

```diff
@@ -123,10 +131,22 @@ function CommitFlamegraph({chartData, commitTree, height, width}: Props) {
     return null;
   }, [chartData, selectedFiberID, selectedChartNodeIndex]);
 
+  const handleElementMouseEnter = ({id, name}) => {
+    highlightNativeElement(id); // Highlight last hovered element.
+    setHoveredFiberData({id, name}); // Set hovered fiber data for tooltip
+  };
+
+  const handleElementMouseLeave = () => {
+    clearHighlightNativeElement(); // clear highlighting of element on mouse leave
+    setHoveredFiberData(null); // clear hovered fiber data for tooltip
+  };
+
   const itemData = useMemo<ItemData>(
     () => ({
       chartData,
-      hoverFiber,
+      isHovered: hoveredFiberData !== null,
+      onElementMouseEnter: handleElementMouseEnter,
+      onElementMouseLeave: handleElementMouseLeave,
```

Comment:
There's no point to `useMemo` if we're passing in functions that get recreated inline every time we render.

Since these functions are just attached as mouse event handlers, normally it would not be important to memoize them- but in this case, not memoizing means that we could be doing a lot of otherwise unnecessary re-renders.